lmdeploy/turbomind/deploy/source_model/llama.py

```diff
@@ -164,7 +170,7 @@ def get_mgrs(self):
                 else:
                     new_params = load_file(osp.join(self.ckpt_path, ckpt))
                 ret = self.Reader(new_params, unused_params,
-                                  i == self.nmgrs - 1, self.model_info())
+                                  i == self.nmgrs - 1, self.model_config)
```

Comment:
This will affect many models, like internlm2, internvl

Reply:
Yes, I know. But I think we need to make the original model config being accessible by all the source models. 
Otherwise, `model_info()` should handle all the corner cases.